projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -816,7 +816,9 @@ public function update_settings() {
 						return new WP_Error( 'invalid_input', __( 'Invalid SEO title format.', 'jetpack' ), 400 );
 					}
 
-					$new_title_formats = Jetpack_SEO_Titles::update_title_formats( $value );
+					$new_title_formats = Jetpack_SEO_Titles::update_title_formats(
+						Jetpack_SEO_Titles::sanitize_title_formats( $value )
```

Comment:
Maybe this could happen directly within `Jetpack_SEO_Titles::update_title_formats` and not as a new method?

Reply:
Refactored for the wpcom side a bit in https://github.com/Automattic/jetpack/pull/18810/commits/fe10ab9e01bc1a22250eff918c26228c3be03124 but the separate `sanitize_title_formats` function is needed to serve the wp-admin side which doesn't call `update_title_formats`